Feature/prop to add rel attribute (#401)
vtex-apps/search-result: docs/README.md

| Name Descending          | `"OrderByNameDESC"`         |

- **`search-fetch-more` block**

> The "Show More" button that is used to load the results of the next search page. This block is not rendered if there is no next page.
> 
> | Prop name       | Type            | Description                                                                                                  | Default value |
> | --------------- | --------------- | ------------------------------------------------------------------------------------------------------------ | ------------- |
> | `relHtmlAttributeForButton` | `[boolean]` | Indicates if `rel='next'` attribute will be added for the Fetch more button. | `false` |

Reviewer: ```suggestion 

The `Show More` button is used to load the results of the next search results page. Even when declared, this block is not rendered if there is no next page.

| Prop name       | Type            | Description                                                                                                  | Default value |
| --------------- | --------------- | ------------------------------------------------------------------------------------------------------------ | ------------- |
| `relHtmlAttributeForButton` | `boolean` | Whether the `rel='next'` attribute should be added to the `Show more` button HTML element (`true`) or not (`false`). | `false` |
```

> | `htmlElementForButton` | `Enum` | Define what type of html element will be rendered for `FetchMoreButton` component. Accept two values: `ANCHOR`, it renders a `<a>` element with `href` and `rel` attributes **OR** `BUTTON`, it renders a `<button>` element without `href` and `rel` attributes | `BUTTON` |

Reviewer: ```suggestion
| `htmlElementForButton` | `enum` | Which HTML element will be displayed for `Show more` button component. Possible values are: `ANCHOR` (displays a `<a>` element with `href` and `rel` attributes)  or `BUTTON` (displays a `<button>` element without `href` and `rel` attributes). | `BUTTON` |
```

Author: @guerreirobeatriz Fixed

> | `htmlElementForFetchMoreButton` | `[enum]` | Define what type of html element will be rendered for `FetchMoreButton` component. Accept two values: `ANCHOR`, it renders a `<a>` element with `href` and `rel` attributes **OR** `BUTTON`, it renders a `<button>` element without `href` and `rel` attributes | `BUTTON` |

Reviewer 2: ```suggestion
| `htmlElementForButton` | `Enum` | Define what type of html element will be rendered for `FetchMoreButton` component. Accept two values: `ANCHOR`, it renders a `<a>` element with `href` and `rel` attributes **OR** `BUTTON`, it renders a `<button>` element without `href` and `rel` attributes | `BUTTON` |
```

- **`search-fetch-previous` block**

> The "Show Previous" button that is used to load the results of the previous search page. This block is not rendered if there is no previous page.
> 
> | Prop name       | Type            | Description                                                                                                  | Default value |
> | --------------- | --------------- | ------------------------------------------------------------------------------------------------------------ | ------------- |
> | `relHtmlAttributeForButton` | `[boolean]` | Indicates if `rel='prev'` attribute will be added for the Fetch previous button. | `false` |

Reviewer: ```suggestion
The `Show Previous` button is used to load the results of the previous search results page. Even when declared, this block is not rendered if there is no previous page.

| Prop name       | Type            | Description                                                                                                  | Default value |
| --------------- | --------------- | ------------------------------------------------------------------------------------------------------------ | ------------- |
| `relHtmlAttributeForButton` | `boolean` | Whether the `rel='prev'` attribute should be added to the `Show previous` button HTML element (`true`) or not (`false`). | `false` |
```

> | Prop name       | Type            | Description                                                                                                  | Default value |
> | --------------- | --------------- | ------------------------------------------------------------------------------------------------------------ | ------------- |
> | `htmlElementForButton` | `Enum` | Define what type of html element will be rendered for `FetchPreviousButton` component. Accept two values: `ANCHOR`, it renders a `<a>` element with `href` and `rel` attributes **OR** `BUTTON`, it renders a `<button>` element without `href` and `rel` attributes | `BUTTON` |

Reviewer: ```suggestion
| Prop name       | Type            | Description                                                                                                  | Default value |
| --------------- | --------------- | ------------------------------------------------------------------------------------------------------------ | ------------- |
| `htmlElementForButton` | `enum` | Which HTML element will be displayed for `Show previous` button component. Possible values are: `ANCHOR` (displays a `<a>` element with `href` and `rel` attributes)  or `BUTTON` (displays a `<button>` element without `href` and `rel` attributes). | `BUTTON` |
```

Author: Fixed

> | `htmlElementForFetchPreviousButton` | `[enum]` | Define what type of html element will be rendered for `FetchPreviousButton` component. Accept two values: `ANCHOR`, it renders a `<a>` element with `href` and `rel` attributes **OR** `BUTTON`, it renders a `<button>` element without `href` and `rel` attributes | `BUTTON` |

Reviewer 2: ```suggestion
| `htmlElementForButton` | `Enum` | Define what type of html element will be rendered for `FetchPreviousButton` component. Accept two values: `ANCHOR`, it renders a `<a>` element with `href` and `rel` attributes **OR** `BUTTON`, it renders a `<button>` element without `href` and `rel` attributes | `BUTTON` |
```
